content/en/docs/v3.5/tutorials/how-to-migrate.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: How to migrate from Etcd2 to Etcd3
-description: Migration guide from Etcd2 to Etcd3
+title: How to migrate etcd from v2 to v3
+description: Migration guide for upgrading etcd from v2 to v3
```

Comment:
How about:

> description: Etcd v2 to v3 migration guide

Reply:
That works for me -- I'll make the change (but with a lowercase `etcd` to start)